docker/apply/config/test.json

```diff
@@ -1,5 +1,5 @@
 {
-  "safelist": ["lists"],
+  "safelist": ["lists", "localhost", "host.docker.internal"],
```

Comment:
I guess you had the NODE_ENV "test" deployed locally.  Probably worth reverting this to minimise the accepted routes to the form runner on the test env.